app/controllers/booking_controller.py

```diff
@@ -0,0 +1,60 @@
+import os
+from . import controllers
+from cache import get_from_cache, set_to_cache
+from config import Config
+from app.utils.dbutil import DBUtil
+from flask import Flask, request, jsonify, render_template
+
+app = Flask('app')
+Config.DATABASE = os.path.join(app.root_path + '/db/', 'database.db')
+app.config.from_object(Config)
+db = DBUtil(app, False).get_instance(False)
```

Comment:
Singletons should not need to be initialized (the constructor should not be called outside of the static get_instance method). I recommend you have DBUtil store the Config object by accessing it globally instead of passing it to the constructor. The end-game is that we should be able to get db singletons in a call like this --> DBUtil.get_instance()

Reply:
These 4 lines do not belong in the booking_controller.py

They should be in __init__.py under app/ , which is the first file run in the app/ module, so in there you'd initialize the app, the database object, etc. to be used across the root module of the app